openpype/hosts/houdini/plugins/publish/collect_redshift_rop.py

```diff
@@ -45,6 +45,25 @@ def process(self, instance):
         beauty_suffix = rop.evalParm("RS_outputBeautyAOVSuffix")
         render_products = []
 
+        # Store whether we are splitting the render job (export + render)
+        split_render = bool(rop.parm("RS_archive_enable").eval())
+        instance.data["splitRender"] = split_render
+        export_prefix = None
+        export_products = []
+        if split_render:
+            export_prefix = evalParmNoFrame(
+                rop, "RS_archive_file", pad_character="0"
+            )
+            beauty_export_product = self.get_render_product_name(
+                prefix=export_prefix,
+                suffix=None)
+            export_products.append(beauty_export_product)
+            self.log.debug(
+                "Found export product: {}".format(beauty_export_product)
+            )
+            instance.data["ifdFile"] = beauty_export_product
```

Comment:
What does ifd stand for? Isn't that a Mantra-specific file whilst the RS_archive_file is **not** actually an IFD it's like a redshift proxy scene right?

As such, the usage of IFD as a name here is quite confusing to me.

Reply:
I agree but since it is used also in the same logic in Arnold collector, I tried not to touch other parts of code. We should rename it in another PR so it is easier to test.

Reply:
Agree - it's a separate refactor this PR if it's erroneous even outside of this PR. We should make sure to track it and don't forget it. :)

Reply:
Also, since this part of the code looks like it is almost the same for every supported renderer, we should yank it out from the individual plugins. As far as I know, only ROP param name is changing.

Reply:
> Agree - it's a separate refactor this PR if it's erroneous even outside of this PR. We should make sure to track it and don't forget it. :)

I'm going to make a PR to clean up this feature. where we 
- can pick a generic name instead of "ifdFile"
- update the job name. 
- change variable name `export_file` to `split_render` other creators for more consistency

Reply:
By the way, `instance.data["exportFiles"]` appears to be unused and thus can be removed.